### ISSUE TITLE
Enchancement Proposal: inactivity mute.

### ISSUE THREAD

**Issue by Ybaedbaenn** [sommd/auto-mute#5]
Hello, I like your idea for the app and its implementation, and I have found two cases where it could be improved.

The first concern is about a situation when a user unmutes a device for listening but does not listen to anything and then forgets to mute it back. In this case, the device will remain unmuted because the app only mutes it after a certain amount of time after listening, not when it is inactive. An option to enable muting after a specific time of inactivity regardless of whether the user was listening would solve this problem.

My second idea is that it should be possible to choose a longer delay for muting after audio stops playing (or after inactivity starts)  than 5 minutes, up to about 30 minutes. People sometimes take short breaks from using the device, for example, for 5 minutes. During that time, they don't need to mute it, so they could set it to stay unmuted. Staying away from it for more than 30 minutes most likely means that the person has already forgotten about listening, and it can be safely muted.

Please let me know what you think of my ideas and if you would be able to add them.

**Comment by sommd** [comment]
Yeah, I've been meaning to add support for auto-muting after manually un-muting for a while now but wasn't sure how useful it'd be so I didn't prioritise it. Good to know that someone would find it useful.

Longer delays is a reasonable ask, it's mostly just a UI problem. Adding more options to the drop-down is not a great solution since there's already too many. I need to implement a better UI but haven't really had the time.

Thanks for the suggestions, when I have time to work on the project again these are probably what I'll work on.
